content/04.workflows-and-software.md

```diff
@@ -28,31 +28,30 @@ Finally, each step becomes a self-contained unit that can be used and re-used ac
 In sum, the internal scaffolding provided by workflow systems helps build analyses that are generally better documented, repeatable, transferable, and scalable.
 
 
-#### Choosing a workflow system
-While the benefits of encoding a workflow in a workflow system are immense, the learning curve associated with implementing complete workflows in a new syntax can be daunting.
+#### Getting started with workflows
+
+**Using existing workflows** While the benefits of encoding a workflow in a workflow system are immense, the learning curve associated with implementing complete workflows in a new syntax can be daunting.
```

Comment:
i like this restructure, but not sure the two headers quite convey what we want yet...